packages/components/src/popover/utils.ts

```diff
@@ -143,8 +143,8 @@ export const placementToMotionAnimationProps = (
  *
  * @param  document The iframe's owner document.
  *
- * @return The offset of the document's frame element, or undefined if the
- * document has no frame element.
+ * @return {Object|undefined} The offset of the document's frame element, or undefined if the
+ *                            document has no frame element.
```

Comment:
no idea why it started to complain here and not with the other blocks in that file :/